internal/pkg/dashboard/apidata/data.go

```diff
@@ -18,6 +18,11 @@ type Data struct {
 	// Data per each node.
 	Nodes map[string]*Node
 
+	// UnresolvedNodes contain the same Node data as Nodes
```

Comment:
is this some kind of a race that we resolve after the graphs start working?

Reply:
Nope, it wasn't a race, the comparison was done wrongly in a consistent way. So when a node alias (name etc.) was used, the graphs were never updating. it went unnoticed.

Also, I now solved the issue in a different way, so this code is no more there.